package/rtcm3_out_protocol/src/Makefile

```diff
@@ -0,0 +1,26 @@
+TARGET=librtcm3_out_protocol
+SOURCES=info_rtcm3_out.c
+CFLAGS=-std=gnu11 -fPIC
```

Comment:
just need to add a plus sign right? i.e. `CFLAGS+=-std=gnu11 -fPIC` etc?

Reply:
NMEA and SBP should have it, all new packages should include our standard battery of warnings too, but since this package has essentially no code it probably isn't a big deal.